### DISCUSSION TITLE
Poor English skills, seeking assistance. 🙏

### DISCUSSION THREAD

**Discussion by tlerbao** [rectorphp/rector#8851] [Q&A] [answered]
For example, the following code will throw an error "Trying to access array offset on value of type null" in PHP 7.4.30.

Since my English is poor, I couldn't find the corresponding upgrade rule to fix this error. Could you please tell me exactly what I should do?
```
    public function requires()
    {
        return $this->content()['require'];
    }
```

**Comment by samsonasik** [comment, marked as answer]
Hi, rector is not error handler, to fix that, you need to create more unit test, activate logger, and validate code with phpstan
